package com.kwad.sdk.core.b.a;

import com.kwad.components.ad.h.a.a.b;
import org.json.JSONObject;

/* loaded from: classes4.dex */
public final class nj implements com.kwad.sdk.core.d<b.a> {
    @Override // com.kwad.sdk.core.d
    public final /* bridge */ /* synthetic */ void a(com.kwad.sdk.core.b bVar, JSONObject jSONObject) {
        a((b.a) bVar, jSONObject);
    }

    @Override // com.kwad.sdk.core.d
    public final /* bridge */ /* synthetic */ JSONObject b(com.kwad.sdk.core.b bVar, JSONObject jSONObject) {
        return b((b.a) bVar, jSONObject);
    }

    private static void a(b.a aVar, JSONObject jSONObject) {
        if (jSONObject == null) {
            return;
        }
        aVar.rn = jSONObject.optInt("timerName");
        aVar.ro = jSONObject.optInt("time");
    }

    private static JSONObject b(b.a aVar, JSONObject jSONObject) {
        if (jSONObject == null) {
            jSONObject = new JSONObject();
        }
        int i2 = aVar.rn;
        if (i2 != 0) {
            com.kwad.sdk.utils.ac.putValue(jSONObject, "timerName", i2);
        }
        int i3 = aVar.ro;
        if (i3 != 0) {
            com.kwad.sdk.utils.ac.putValue(jSONObject, "time", i3);
        }
        return jSONObject;
    }
}
